modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml

```diff
@@ -0,0 +1,135 @@
+uuid: dbbff4a1-e9dc-4942-b91c-6533223d9055
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.